examples/seismic/tti/wavesolver.py

```diff
@@ -126,3 +133,78 @@ def forward(self, src=None, rec=None, u=None, v=None, vp=None,
         summary = op.apply(src=src, rec=rec, u=u, v=v,
                            dt=kwargs.pop('dt', self.dt), **kwargs)
         return rec, u, v, summary
+
+    def adjoint(self, rec, srca=None, u=None, v=None, vp=None,
+                epsilon=None, delta=None, theta=None, phi=None,
+                save=False, kernel='centered', **kwargs):
+        """
+        Forward modelling function that creates the necessary
```

Comment:
Description needs to be changed, right?

Reply:
yes, the dangers of copy-paste practice :P